### PR TITLE
Expose query string paraemters not only for GET requests.

### PR DESCRIPTION
**Pull request by ar45**
query string parameters are set as request_variables and subsequently overridden by post variables.

- `request.all()` and `request.input()` return combined query params and post params.
- Added method `request.query_param(name, default, multi_value=False)` to retrieve query string parameters.
- CONTENT_TYPE case insensitive

### CODE REVIEW

File: src/masonite/request.py [hunk 1]
```diff
@@ -89,6 +91,29 @@ def input(self, name, default=False, clean=False, quote=True):
...
+        Keyword Arguments:
+            default {any} -- Default value if input does not exist (default: {None})
+            first {bool} -- Whether to return all values of a multi value query string param
+                            ie. quuery_param=one&query_param=two (default: {False})
+
+        Returns:
+            any
+        """
+        try:
+            value = self.query_params[name]
```
Comment (josephmancuso) [inline src/masonite/request.py:109]: This block can simply go to `value = self.query_params.get(name, default)`

Comment (ar45) [reply src/masonite/request.py:109]: If not multi_value, we return value [0] which is not what we want to do for default [0]. default is to be returned verbatim.

**Comment by ar45** [comment]
I still have to add some tests for this.

**Comment by josephmancuso** [comment]
I added a way to get all query params

**Comment by ar45** [comment]
Thanks @josephmancuso

**Pull request merged**